drivers/overlay/overlay.go

```diff
@@ -124,10 +123,6 @@ func Init(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 		return nil, err
 	}
 
-	if err := supportsOverlay(); err != nil {
-		return nil, errors.Wrap(graphdriver.ErrNotSupported, "kernel does not support overlay fs")
-	}
-
 	// require kernel 4.0.0 to ensure multiple lower dirs are supported
 	v, err := kernel.GetKernelVersion()
```

Comment:
Shold we get rid of this kernel version check also. This will not work with rhel kernels. In fact, while you are doing overlay tests, you can try to mount multiple lower directories and if that works, you know overlay is supported at the same time kernel is new enough to support multiple lower and there should not be any need for version check.

Reply:
That's a good point.  I'll drop this bit.